frame/contracts/src/wasm/mod.rs

```diff
@@ -1430,7 +1430,7 @@ mod tests {
 				MockExt::default(),
 				&mut gas_meter
 			),
-			Err(ExecError { reason: DispatchError::Other("during execution"), buffer: _ })
+			Err(ExecError { reason: DispatchError::Other("contract trapped during execution"), buffer: _ })
```

Comment:
Can we wrap this and the line below, since they exceed 100 characters soft-limit?

Reply:
done